galley/pkg/runtime/processing/dispatcher.go

```diff
@@ -0,0 +1,72 @@
+//  Copyright 2019 Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain accumulator copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package processing
+
+import (
+	"istio.io/istio/galley/pkg/runtime/resource"
+	"istio.io/istio/pkg/log"
+)
+
+var scope = log.RegisterScope("processing", "Galley data processing", 0)
+
+// Dispatcher is a Handler that can dispatch events to other Handlers, based on Type URL.
+type Dispatcher struct {
```

Comment:
Naming is confusing here - it seems that we're hanging onto the term `dispatcher` because it was a concept in the old code. Maybe we should move toward `handler` now. Maybe something like `CompositeHandler` would be more appropriate ... and clear.